### ISSUE TITLE
Focus on headerfilter

### ISSUE THREAD

**Issue by leonardorame** [olifolkerd/tabulator#769]
Hi Oli, I cannot seem to find a component method to focus on a header filter.

What I would like to do is to automatically focus on the 1nst column header filter after the `tableBuilt` event.

Leonardo.

**Comment by olifolkerd** [comment]
Hey Leonardo, there is currently no way to do this inbuilt in Tabulator, however i like the idea so will add it to the road map.

In the mean time could i suggest using a custom header filter that you set an id on the filter element so you can select it by id in the ***tableBuilt*** event and then focus on it.

**Comment by olifolkerd** [comment]
Hey @leonardorame 

You will be happy to hear i have just pushed a commit with this functionality to the 3.4 branch.

The new ***setHeaderFilterFocus*** function will allow you to set focus on a header element:

```js
$("#example-table").tabulator("setHeaderFilterFocus", "name"); //focus on the header filter for the name column
```

This will be included in the 3.4 release in a couple of weeks time.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
Tabulator version 3.4 has now been officially released!

You can find details of the new release in the [News Section](http://tabulator.info/news)

Full documentation of all the new features can be found [Here](http://tabulator.info/docs/3.4)

Cheers

Oli :)
